<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbstractPlaybackFile</class>
 <widget class="QWidget" name="AbstractPlaybackFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_page">
       <property name="styleSheet">
        <string notr="true">min-height: 0px;</string>
       </property>
       <property name="text">
        <string>Total: Page:</string>
       </property>
       <property name="style" stdset="0">
        <string>transparent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_firstPage">
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/startpage_g.png);
}
QToolButton:hover
{
	background-color: transparent;
	border-image: url(:/playback/playback/startpage_b.png);
}
QToolButton:pressed
{
	background-color: transparent;
	border-image: url(:/playback/playback/startpage_g.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_previousPage">
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/prevpage_g.png);
}
QToolButton:hover
{
	background-color: transparent;
	border-image: url(:/playback/playback/prevpage_b.png);
}
QToolButton:pressed
{
	background-color: transparent;
	border-image: url(:/playback/playback/prevpage_g.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyLineEditPage" name="lineEdit_page">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_nextPage">
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/nextpage_g.png);
}
QToolButton:hover
{
	background-color: transparent;
	border-image: url(:/playback/playback/nextpage_b.png);
}
QToolButton:pressed
{
	background-color: transparent;
	border-image: url(:/playback/playback/nextpage_g.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_lastPage">
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/endpage_g.png);
}
QToolButton:hover
{
	background-color: transparent;
	border-image: url(:/playback/playback/endpage_b.png);
}
QToolButton:pressed
{
	background-color: transparent;
	border-image: url(:/playback/playback/endpage_g.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButtonPage" name="pushButton_go">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_selectedCount">
        <property name="styleSheet">
         <string notr="true">min-height: 0px;</string>
        </property>
        <property name="text">
         <string>Select Items: 0</string>
        </property>
        <property name="style" stdset="0">
         <string>transparent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_selectedSize">
        <property name="styleSheet">
         <string notr="true">min-height: 0px;</string>
        </property>
        <property name="text">
         <string>Total Size: 0</string>
        </property>
        <property name="style" stdset="0">
         <string>transparent</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEditPage</class>
   <extends>QLineEdit</extends>
   <header>mylineeditpage.h</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>tableview.h</header>
  </customwidget>
  <customwidget>
   <class>PushButtonPage</class>
   <extends>QPushButton</extends>
   <header>PushButtonPage.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
